installer/opensuse/prepare

```diff
@@ -0,0 +1,129 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This is a distro-specific continuation of the prepare.sh script.
+
+PKGEXT='rpm'
+DISTROAKA='opensuse'
+
+
+# install_dist: see install() in prepare.sh for details.
+install_dist() {
+    local pkgs='' params='' asdeps=''
+    while [ "$#" != 0 ]; do
+        if [ "$1" = "--minimal" ]; then
+            params='--no-recommends'
+        elif [ "$1" = "--asdeps" ]; then
+            shift
+	    continue
+        else
+            break
+        fi
+        shift
+    done
+    while [ "$#" != 0 ]; do
+        if [ "$1" = '--' ]; then
+            shift
+            break
+        fi
+        pkgs="$pkgs $1"
+       shift
+    done
+    zypper -q install -y $params $pkgs `list_uninstalled_dist - "$@"` || return $?
+}
+
+
+# install_pkg_dist: see install_pkg() in prepare.sh for details.
+install_pkg_dist() {
+    local params=''
+    if [ "$1" = '--minimal' ]; then
+        params='--no-recommends'
+        shift
+    fi
+    if [ "$#" != 0 ]; then
+        zypper -q install -y $params "$@" || return $?
+    fi
+}
+
+
+# install_dummy_dist: see install_dummy() in prepare.sh for details.
+install_dummy_dist() {
+    if [ "$#" = 0 ]; then
+        return
+    fi
+    local pkgname="crouton-$1" pkgprovides="$1" requires=""
+    shift
+    while [ "$#" != 0 ]; do
+        if [ "$1" = '--' ]; then
+            shift
+            break
+        fi
+        pkgprovides="$pkgprovides, $1"
+        shift
+    done
+    local pkgdepends="$1"
+    if [ "$#" != 0 ]; then
+        shift
+        while [ "$#" != 0 ]; do
+            pkgdepends="$pkgdepends, $1"
+            shift
+        done
+    fi
+
+    if [ $pkgdepends ]; then
+        requires="Requires: ${pkgdepends}"
+    fi
+
+    local tmp="`mktemp -d crouton.XXXXXX --tmpdir=/tmp`"
+    addtrap "rm -rf '$tmp'"
+    cat > "$tmp/dummy.spec" <<EOF
+Name: $pkgname
+Version: 1.0.0
+Release: 1.0.0
+Group: dummy
+Vendor: dummy
+License: %{vendor}
+Provides: $pkgprovides
+$requires
+Summary: Provides a dummy ${pkgname#*-} for crouton
+BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+
+%description
+%{summary}
+
+%files
+EOF
+    rpmbuild --define='_rpmdir "$tmp"' -bb "$tmp"/dummy.spec > "$tmp"/build.log
```

Comment:
quotes should surround the whole string (e.g., `"$tmp/dummy.spec"`) as long as wildcards aren't involved